templates/branding/openSUSE/external_reporting.html.ep

```diff
@@ -7,12 +7,14 @@
     sle => 'https://bugzilla.suse.com/enter_bug.cgi',
     opensuse => 'https://bugzilla.opensuse.org/enter_bug.cgi',
     casp => 'https://bugzilla.suse.com/enter_bug.cgi',
+    caasp => 'https://bugzilla.suse.com/enter_bug.cgi',
```

Comment:
there is no casp anymore, so just add the 2nd a to it

Reply:
it's not "just" an "a" but also a different name on bugzilla hence the other changes. So updated the commit now to fully replace the old "casp" handling with corresponding "caasp" handling